assets/src/edit-story/components/library/panes/shared/libraryMoveable.js

```diff
@@ -76,10 +77,15 @@ function LibraryMoveable({
   const overlayRef = useRef(null);
   const moveable = useRef(null);
 
+  // These useLayout's are specifically kept as separate entries
```

Comment:
Hm, what would happen when using the same `useLayout`?

Reply:
Nothing out of the ordinary, it's just the same function invoked twice with different arguments.